docs/src/pages/components/authenticator/quick-start.react.mdx

```diff
@@ -1,6 +1,23 @@
-For React, import and wrap your app using the `withAuthenticator`
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+For React, you can use the `Authenticator` component directly, or wrap your app in `withAuthenticator`
 [Higher-Order Component](https://reactjs.org/docs/higher-order-components.html):
 
+<Tabs>
+  <TabItem title="Authenticator">
+
+```js{3-4,11-12} file=../../../../../examples/next/pages/ui/components/authenticator/sign-in-with-username/index.page.tsx
+
+```
+
+  </TabItem>
+  <TabItem title="withAuthenticator">
+
+The `withAuthenticator` automatically sets the [`variation`](#variation) to `modal`.
```

Comment:
I was going to ask you if you should document what `withAuthenticator` uses for its variation, but you're already ahead of me 😄

Reply:
There's an open issue (#640) to prefer `Authenticator` over `withAuthenticator`, so this was the 1st step towards that goal 💪